src/IdePurescript/Modules.purs

```diff
@@ -116,17 +117,26 @@ mkImplicit :: String -> Module
 mkImplicit m = Module
   { qualifier: Nothing, importType: Implicit, moduleName: m }
 
+-- Returns unqualified modules that can possibly import the ident. If ident is
+-- Nothing it returns only implicit imports.
 getUnqualActiveModules :: State -> Maybe String -> Array String
 getUnqualActiveModules state ident = getModules include state
   where
   include (Module { qualifier: Just _ }) = false
-  include (Module { importType: Explicit idents }) = maybe false
-    (\x -> x `elem` idents || ("(" <> x <> ")") `elem` idents)
```

Comment:
I believe the `()` check is related to operators or suchlike, why is this being changed? Is there a specific case that seems to be handled incorrectly?

Regarding the constructor import comment, that is something that will be addressed shortly when I revive the changes for recent `purs` versions, where mostly this code should be less required.